### PR TITLE
Remove list-styling from error summary

### PR DESCRIPTION
**Pull request by saarasat**
This removes the list-styling and bullets from the
error summary component.

Related to changes in: https://github.com/apluslms/a-plus-design-system/pull/9
Also the only implementation of this component so far, has been matched to this style in:
https://github.com/apluslms/mooc-grader/pull/82

# Testing

Outlook has been tested with multiple and only one erroneous field:

image

image

# Have you updated the README or other relevant documentation?

Not relevant

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

*Clean up your git commit history before submitting the pull request!*

- [ ] I (developer) have created unit tests and the tests pass
- [ ] I (developer) have created functional tests (Selenium tests) if applicable
- [x] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

**Review by mtthwhggns — APPROVED** [review]
This looks very simple and the screenshots look good.

File: assets/css/main.css [hunk 1]
```diff
@@ -1,9 +1,8276 @@
-﻿/*!
```
Comment (Mankro) [inline assets/css/main.css:1]: Is this intentional that `main.css` seems to contain the whole Bootstrap CSS? This is not the built minimized output of SASS.

**Review by Mankro — CHANGES_REQUESTED** [review]
Looks good but I get the feeling that the CSS has been compiled in a wrong way.

File: assets/css/main.css (hunk 1 above)
Comment (saarasat) [reply assets/css/main.css]: That is weird, I thought I ran `drone exec`. Apparently committed at the wrong phase then. In any case, thanks for pointing out, this has now been fixed!

Comment (raphendyr) [reply assets/css/main.css]: `main.css` always contains the bootstrap stuff, but that should be in a single line normally. It sometimes could happen that one runs drone exec, but has the sass watch container running at the same time, and then they touch a source file, which triggers a rebuild, which overwrites the `main.css` again. One thing is to run `git diff --cached` to verify that you are committing what you think you are. However, `main.css` doesn't format there so well.

Comment (saarasat) [reply assets/css/main.css]: Good to know!

**Review by raphendyr — APPROVED** [review]

**Review by Mankro — APPROVED** [review]
I rebased and updated the compiled CSS output in order to resolve the git conflict (and force-pushed).

**Pull request merged**